lib/core/virtual_host_scanner.py

```diff
@@ -6,9 +6,21 @@
 import pandas as pd
 import time
 from lib.core.discovered_host import *
+from urllib3.util import ssl_
 
 DEFAULT_USER_AGENT = 'Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36'
 
+_target_host = None
+_ssl_wrap_socket = ssl_.ssl_wrap_socket
+def ssl_wrap_socket(sock, keyfile=None, certfile=None, cert_reqs=None,
```

Comment:
I think we should move this into the class. Going to merge for now since it's a bugfix but I think we should definitely do some refactoring on this file soon.